### ISSUE TITLE
[SfLinearGauge] _barPointerOffset@569180565' has not been initialized.

### ISSUE THREAD

**Issue by monolidth** [syncfusion/flutter-widgets#910]
Hi, 
I face a problem regarding the SfLinearGauge.

For me it looks like the paint function was never called and therefore the `_barPointerOffset = offset;` was never executed.

Minimum example:
```
SfLinearGauge(
                  minimum: low,
                  maximum: high,
                  showLabels: false,
                  showAxisTrack: false,
                  showTicks: false,
                  labelOffset: 0,
                  tickOffset: 0,
                  barPointers: [
                      LinearBarPointer(
                          value: high,
                          thickness: thickness,
                          edgeStyle: LinearEdgeStyle.bothCurve,
                          color: Theme.of(context).dividerColor),
                    ],
                  markerPointers: [
                      if (score != 0)
                        LinearShapePointer(
                          elevation: 4,
                          elevationColor: getColor(context),
                          enableAnimation: true,
                          width: 20.w,
                          height: thickness,
                          color: green,
                          value: score,
                          shapeType: LinearShapePointerType.rectangle,
                          position: LinearElementPosition.cross,
                        )
                    ]),
```

Trace:
```
The relevant error-causing widget was:  SfLinearGauge When the exception was thrown, this was the stack: 
#0      RenderLinearBarPointer._barPointerOffset (package:syncfusion_flutter_gauges/src/linear_gauge/pointers/linear_bar_renderer.dart)
#1      RenderLinearBarPointer.value= (package:syncfusion_flutter_gauges/src/linear_gauge/pointers/linear_bar_renderer.dart:71:11)
#2      LinearBarPointer.updateRenderObject (package:syncfusion_flutter_gauges/src/linear_gauge/pointers/linear_bar_pointer.dart:296:9)
#3      RenderObjectElement._performRebuild (package:flutter/src/widgets/framework.dart:5784:36)
#4      RenderObjectElement.update (package:flutter/src/widgets/framework.dart:5763:5)
#5      SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:6305:11)
#6      Element.updateChild (package:flutter/src/widgets/framework.dart:3570:15)
#7      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4904:16)
```

Note:
I strongly recommend an issue template for bugs and features. 

Regards,

William Todt

**Comment by SriramKiranSenthilkumar** [comment]
Hi @monolidth,

Greetings from Syncfusion. We have checked your query and we would like to let you know that we tried to replicate the reported issue at our end by referring to your code snippet at our end. Unfortunately, we didn't face any exception reported and so the issue didn't get reproduced. We have tried the following cases for replicating the issue at our end.
1.	By adding interaction for dragging the pointers in the linear gauge.
2.	By providing higher animation duration value for the bar pointer.

As we are not exactly sure on what scenario the issue replicates, we request you to kindly try to replicate the reported issue in the below-attached test sample and revert us so that it will help us assist you in a better way.

Sample: [gauge_410461.zip](https://github.com/syncfusion/flutter-widgets/files/9704591/gauge_410461.zip)


Regards,
Sriram Kiran
